packages/react-button/package.json

```diff
@@ -14,26 +14,23 @@
   },
   "license": "MIT",
   "scripts": {
-    "build": "just-scripts build",
-    "bundle": "just-scripts bundle",
-    "clean": "just-scripts clean",
-    "code-style": "just-scripts code-style",
-    "just": "just-scripts",
-    "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook",
-    "start:legacy": "just-scripts dev",
-    "start-test": "just-scripts jest-watch",
-    "test": "just-scripts test",
-    "update-api": "just-scripts update-api",
-    "update-snapshots": "just-scripts jest -u"
+    "build": "echo 'TODO: rollup & tsc --declaration && api-extractor'",
+    "clean": "rm -rf dist lib-{amd,commonjs,es-2015} .cache",
+    "start": "yarn serve",
```

Comment:
I suspect that isn't expected to work yet. Also a bit surprised `yarn workspace @fluentui/react-button start` even works since react-button is now excluded from the workspace config.

Reply:
So, does that mean once the whole dependency tree (of converged components and utilities) uses TS path aliases this will work?

Is there something else we have to do to fix the problem where storybook resolves the `"module"` field?

Reply:
> Is there something else we have to do to fix the problem where storybook resolves the "module" field?

I guess we need to remove those hardcoded fields and generate them only during build/publish (or point them to source instead of `lib`)